ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/ClusterDetailsTable.tsx

```diff
@@ -0,0 +1,119 @@
+                        <Td modifier="fitContent" width={10}>
+                            Controls
+                        </Td>
+                        <Th modifier="fitContent" width={10}>
+                            Compliance status
+                        </Th>
+                    </Tr>
+                </Thead>
+                <TbodyUnified
+                    tableState={tableState}
+                    colSpan={3}
+                    errorProps={{
+                        title: 'There was an error loading results for this cluster',
+                    }}
+                    emptyProps={{
+                        message: 'No results found for this cluster',
+                    }}
+                    filteredEmptyProps={{
+                        title: 'No results found',
+                        message: 'Clear all filters and try again',
+                    }}
+                    renderer={({ data }) => (
+                        <Tbody>
+                            {data.map((checkResult) => {
+                                const { checkName, rationale, status } = checkResult;
+                                const clusterStatusObject = getClusterResultsStatusObject(status);
+
+                                return (
+                                    <Tr key={clusterId}>
+                                        <Td dataLabel="Check">
+                                            <Link
+                                                to={generatePath(coverageCheckDetailsPath, {
+                                                    checkName,
+                                                    profileName,
+                                                })}
+                                            >
+                                                {checkName}
+                                            </Link>
+                                            {/*
+                                                grid display is required to prevent the cell from
+                                                expanding to the text length. The Truncate PF component
+                                                is not used here because it displays a tooltip on hover
+                                            */}
+                                            <div style={{ display: 'grid' }}>
+                                                <Text
+                                                    component={TextVariants.small}
+                                                    className="pf-v5-u-color-200 pf-v5-u-text-truncate"
+                                                >
+                                                    {rationale}
+                                                </Text>
+                                            </div>
+                                        </Td>
+                                        <Td dataLabel="Controls">placeholder</Td>
```

Comment:
Is this provided in the `v2/compliance/scan/results/profiles/:profileName/clusters/:clusterID` API result? I noticed the `control` field but it was always empty.

Reply:
Good comments, a lot the questions you have were mentioned in the notes. I can expand on them more if you still have questions.